content/third-party-overlay/third-party-overlays.md

```diff
@@ -0,0 +1,6 @@
+# Third Party Overlays
+
+You can also create your own overlays. Overlays will be passed the
+following props [transparently](https://
+github.com/uber/react-map-gl/blob/master/src/map.react.js#L589) 
+provided they're a direct children of a `<MapGL/>` map.
```

Comment:
One thing I forgot to mention is that the docs also need to be updated to the new, 0.6 API so for example, it's no longer the case that overlays have to be direct children of the `<MapGL/>` element.